src/Rubric/RubSmalltalkEditor.class.st

```diff
@@ -96,7 +96,7 @@ RubSmalltalkEditor class >> buildShortcutsOn: aBuilder [
 		do: [ :target | target editor jumpToNextKeywordOfIt: true ]
 		description: 'Jump to next keyword'.
 		
-	(aBuilder shortcut: #jumpToNextKeywordOfIt)
+	(aBuilder shortcut: #jumpToPrevKeywordOfIt)
 		category: RubSmalltalkEditor name
 		default: $j shift meta
 		do: [ :target | target editor jumpToNextKeywordOfIt: false ]
```

Comment:
Here it is not consistent with the editor message.

Reply:
yes, the boolean changes it into a backjump. Bad method name... 

jumpToNextKeywordOfIt: isForwardJump "Jump to the next/previous keyword after the cursor"

Reply:
ah :-)